build/images/base/Dockerfile.ubi

```diff
@@ -33,7 +34,10 @@ RUN set -eux; \
          *) pluginsArch=''; echo >&2; echo >&2 "unsupported architecture '$dpkgArch'"; echo >&2 ; exit 1 ;; \
     esac; \
     mkdir -p /opt/cni/bin; \
-    wget -q -O - https://github.com/containernetworking/plugins/releases/download/$CNI_BINARIES_VERSION/cni-plugins-linux-${pluginsArch}-$CNI_BINARIES_VERSION.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS
+    wget -q -O - https://github.com/containernetworking/plugins/releases/download/$CNI_BINARIES_VERSION/cni-plugins-linux-${pluginsArch}-$CNI_BINARIES_VERSION.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS; \
+    mkdir -p /opt/cni/bin/bgp; \
+    wget -q -O - https://github.com/osrg/gobgp/releases/download/v$BGP_VERSION/gobgp_${BGP_VERSION}_linux_${pluginsArch}.tar.gz | tar xz -C /opt/cni/bin/bgp; \
+    find /opt/cni/bin/bgp -type f ! -name 'gobgp' -delete    
```

Comment:
This should be its own build stage, not merged into the `cni-binaries` build stage